### ISSUE TITLE
@bufbuild/connect-web -> @connectrpc/connect-web

### ISSUE THREAD

**Issue by robertvolkmann** [metal-stack-cloud/api#146]
@bufbuild/connect-web was deprecated. We have to use the new location.

See: https://github.com/connectrpc/connect-es/releases/tag/v0.13.1

**Comment by majst01** [comment]
Yes, you are right, i am aware of this but was not able to migrate `buf.gen.yaml` to generate the ts/js code to use `connectrpc/connect-es` instead.

Maybe we could fix this together this week
